cli/Makefile

```diff
@@ -23,7 +23,7 @@ build: ## Build mizu CLI binary (select platform via GOOS / GOARCH env variables
 					   -X 'github.com/up9inc/mizu/cli/mizu.Branch=$(GIT_BRANCH)' \
 					   -X 'github.com/up9inc/mizu/cli/mizu.BuildTimestamp=$(BUILD_TIMESTAMP)' \
 					   -X 'github.com/up9inc/mizu/cli/mizu.Platform=$(SUFFIX)' \
-					   -X 'github.com/up9inc/mizu/cli/mizu.SemVer=$(SEM_VER)'" \
+					   -X 'github.com/up9inc/mizu/cli/mizu.Ver=$(SEM_VER)'" \
```

Comment:
SEM_VER variable is 0.0.0, we want it to stay this way?

Reply:
it should be changed to 0.0 or 0.1 IMO